Source/WebCore/rendering/TextPainter.h

```diff
@@ -18,14 +18,14 @@
  * along with this library; see the file COPYING.LIB.  If not, write to
  * the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
  * Boston, MA 02110-1301, USA.
- *
  */
 
 #pragma once
 
 #include "AffineTransform.h"
 #include "FloatRect.h"
 #include "GlyphDisplayListCache.h"
+#include "RenderCombineText.h"
```

Comment:
This might regress compile times. I suspect that instead you need to include CheckedPtr if that's what you get an error for.